fe/fe-core/src/main/java/org/apache/doris/catalog/TabletInvertedIndex.java

```diff
@@ -361,7 +358,7 @@ private boolean needRecover(Replica replicaInFe, int schemaHashInFe, TTabletInfo
             return false;
         }
 
-        if (replicaInFe.getVersionHash() == 0 && backendTabletInfo.getVersion() == replicaInFe.getVersion() - 1) {
+        if (backendTabletInfo.getVersion() == replicaInFe.getVersion() - 1) {
```

Comment:
According to the comment below, I think we need to double check this case.

Reply:
it is useless, I remove this if branch.